apps/client/src/components/molecules/button/Button.component.tsx

```diff
@@ -64,9 +71,12 @@ export function Button({
           }
         />
       ) : null}
-      <Text type='Label' size='Medium' className='truncate'>
-        {children}
-      </Text>
+      <div className='flex items-center flex-grow justify-center gap-10'>
+        {input ? input : null}
```

Comment:
Do we need the ternary operator here ?

Reply:
Fixed in 689a160452b9e66a9cf28039c102bcb3248d76bb (PR #52)